awx/api/views/__init__.py

```diff
@@ -530,8 +530,6 @@ def get(self, request):
         auth_backends.sort(key=lambda x: 'g' if x[0] == 'google-oauth2' else x[0])
         for name, backend in auth_backends:
             if (not feature_exists('enterprise_auth') and
```

Comment:
I think we can lose this entire block with the `continue` because the point is to _skip_ certain auth backends if enterprise auth is off.

Reply:
Oof, this `if` statement is painful to unpack.

I think what we *actually* want here is something like this (@cmey

Reply:
Checked with @chrismeyersfsu and @AlanCoding , the change has been made!